### PR TITLE
Search sdk v2

### PR DESCRIPTION
**Pull request by divyadaglia-unbxd**
- create sym link for UnbxdSearch_v2
- Set up demo for v2 search

### CODE REVIEW

File: unbxdSearch.js [hunk 1]
```diff
@@ -382,6 +388,64 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
         return txt.trim().length > 0 ? txt : "&nbsp;&nbsp;&nbsp;";
     });
 
+    Handlebars.registerHelper("ifGrid", function (viewType, options) {
```
Comment (Anupama-Unbxd) [inline unbxdSearch.js:391]: can this also be documented in the google docs please

File: unbxdSearch.js [hunk 2]
```diff
@@ -440,8 +502,58 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
...
+            //     event.preventDefault()
+
+            //     slider_min = window.unbxd_min;
+            //     slider_max = window.unbxd_max;
+
+            //     searchobj
+            //         .clearRangeFiltes()
+            //         .setPage(1)
+            //         .callResults(searchobj.paintResultSet, true);
+            // })
```
Comment (Anupama-Unbxd) [inline unbxdSearch.js:552]: is this function needed if it is all commented out?

File: unbxdSearch.js [hunk 3]
```diff
@@ -440,8 +502,58 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
...
+        },
+        setDefaultFilters: function() {
+           
+        },
```
Comment (Anupama-Unbxd) [inline unbxdSearch.js:556]: is this empty function needed?

Comment (divyadaglia-unbxd) [reply unbxdSearch.js:556]: This is part of default config, to show that this option is available. Just like how we have set fields to empty array to indicate that config existence.

File: unbxdSearch.js (hunk 2 above)
Comment (divyadaglia-unbxd) [reply unbxdSearch.js:552]: I've kept it incase some customer asks for slider implementation, for code reference.
Can be removed.

**Pull request merged**